Source/DSPSpy/main_spy.cpp

```diff
@@ -69,58 +68,15 @@ u16 dspreg_in[32] = {
     0x0000, 0x0000, 0x0003, 0x0004, 0x8000, 0x000C, 0x0007, 0x0008, 0x0009, 0x000a,
 };  ///            ax_h_1   ax_h_1
 
-/* ttt ?
-
-u16 dspreg_in[32] = {
-0x0e4c, 0x03c0, 0x0bd9, 0x06a3, 0x0c06, 0x0240, 0x0010, 0x0ecc,
-0x0000, 0x0000, 0x0000, 0x0000, 0x0322, 0x0000, 0x0000, 0x0000,
-0x0000, 0x0000, 0x00ff, 0x1b41, 0x0000, 0x0040, 0x00ff, 0x0000,
-0x1000, 0x96cc, 0x0000, 0x0000, 0x3fc0, 0x96cc, 0x0000, 0x0000,
-}; */
-
-// if i set bit 0x4000 of SR my tests crashes :(
-
-/*
-// zelda 0x00da
-u16 dspreg_in[32] = {
-0x0a50, 0x0ca2, 0x04f8, 0x0ab0, 0x8039, 0x0000, 0x0000, 0x0000,
-0xFFFF, 0xFFFF, 0xFFFF, 0xFFFF, 0x03d1, 0x0000, 0x0418, 0x0002,     // r08 must have a value ... no
-idea why (ector: it's the looped addressing regs)
-0x0000, 0x0000, 0x00ff, 0x1804, 0xdb70, 0x4ddb, 0x0000, 0x0000,
-0x0000, 0x0000, 0x0000, 0xde6d, 0x0000, 0x0000, 0x0000, 0x004e,
-};*/
-
 u16 dspreg_out[1000][32];
 
-/*
-// gba ucode dmas result here
-u32 SecParams_out[2] __attribute__ ((aligned (0x20))) = {
-  0x11223344, // key
-  0x55667788 // bootinfo
-};
-
-// ripped from demo
-u32 SecParams_in[8] __attribute__ ((aligned (0x20))) = {
-  0xDB967E0F, // key from gba
-  0x00000002,
-  0x00000002,
-  0x00001078,
-  (u32)SecParams_out, //0x80075060, // ptr to receiving buffer
-  // padding?
-  0x00000000,
-  0x00000000,
-  0x00000000
-};
-*/
-
-// UI (interactive register editing)
+// UI (interactive register editing.)
 u32 ui_mode;
 
 enum
 {
   UIM_SEL = 1,
   UIM_EDIT_REG = 2,
-  UIM_EDIT_BIN = 4,
 };
```

Comment:
I've added this change, along with swapping out all of the `UIM_SEL` and `UIM_EDIT_REG` to `UIRegisterMode::Select` and `UIRegisterMode::Edit` respectively. 

Also, I'm not entirely sure why the original author gave the previous values seemingly arbitrary values of 1, 2, and 4, but just in case they had a good reason I've keep these values the same in the rewritten version.